Mathlib/MeasureTheory/Integral/Lebesgue/Basic.lean

```diff
@@ -779,14 +827,18 @@ theorem lintegral_add_mul_meas_add_le_le_lintegral {f g : α → ℝ≥0∞} (hl
   simp only [indicator_apply]; split_ifs with hx₂
   exacts [hx₂, (add_zero _).trans_le <| (hφ_le x).trans hx₁]
 
-/-- **Markov's inequality** also known as **Chebyshev's first inequality**. -/
+/-- **Markov's inequality** aka **Chebyshev's first inequality**.
+Multiplication form for `AEMeasurable` functions. -/
 theorem mul_meas_ge_le_lintegral₀ {f : α → ℝ≥0∞} (hf : AEMeasurable f μ) (ε : ℝ≥0∞) :
-    ε * μ { x | ε ≤ f x } ≤ ∫⁻ a, f a ∂μ := by
-  simpa only [lintegral_zero, zero_add] using
-    lintegral_add_mul_meas_add_le_le_lintegral (ae_of_all _ fun x => zero_le (f x)) hf ε
+    ε * μ { x | ε ≤ f x } ≤ ∫⁻ a, f a ∂μ :=
+  calc
```

Comment:
Not sure why you want to switch to a calc block instead of the shorter current proof. But having the statement with `≤` and the proof with `≥` looks really confusing.